apps/frontend/src/pages/profile/profile.tsx

```diff
@@ -73,6 +74,24 @@ const Profile: React.FC = () => {
 
 	const isLoading = dataStatus === DataStatus.PENDING;
 
+	const [isDirty, setIsDirty] = useState<boolean>(false);
```

Comment:
Not sure to understand why we need this state. `react-hook-form` should handle `isDirty`

Reply:
We need this state to handle two forms in separate components, but `react-router-dom` allows only one blocker per route. For example, after the user changes something in the form of the `UpdateUserForm` component, the blocker and router don't work and the user gets this warning in the console
![image](https://github.com/user-attachments/assets/9ae6a4e5-a552-4405-b6e5-2bb3445ee7d0)